Anytype/Sources/PresentationLayer/Modules/ObjectType/ObjectTypeViewModel.swift

```diff
@@ -95,6 +103,16 @@ final class ObjectTypeViewModel: ObservableObject, RelationsListModuleOutput {
         }
     }
     
+    func onDeleteTap() {
+        showDeleteConfirmation.toggle()
+    }
+    
+    func onDeleteConfirm() async throws {
+        try await objectActionsService.delete(objectIds: [document.objectId])
+        showDeleteConfirmation = false
```

Comment:
nit
you can `dismiss` in `TypeDeleteAlert` and don't handle `showDeleteConfirmation` here